src/picker/clipboarder/pbcopy.py

```diff
@@ -0,0 +1,28 @@
+from subprocess import run
+
+from ..abstractionhelper import is_installed, is_macos
+from ..typer.typer import Typer
+from .clipboarder import Clipboarder
+
+
+class PBCopyClipboarder(Clipboarder):
+    @staticmethod
+    def supported() -> bool:
+        return is_macos() and is_installed("pbcopy")
```

Comment:
Do we need `is_macos`? Is it even possible to have `pbcopy` installed on Linux?

Reply:
That sounds sensible. So no `is_macos()` for `pbcopy` and `cliclick`, but `is_macos() and is_installed('choose')`.